crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,10 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        return false if file.basename.to_s =~ /^template-(.*).json$/
```

Comment:
Can you add back the `file.to_s =~ /knife\/databags\/crowbar/` check? This is definitely a requirement. And then, with the check of the content of the data bag item, I think we're good.

+1 after that.